Automattic/jetpack
# Instant Search: setup preact-testing-library

Sets up automated testing for our Instant Search Preact app.

https://github.com/testing-library/preact-testing-library#docs

#### Changes proposed in this Pull Request:

* No functional changes.

#### Is this a new feature or does it add/remove features to an existing part of Jetpack?
* No.

#### Testing instructions:






Try `yarn test-search` and `yarn test-extensions`, and ensure that the tests pass.

#### Proposed changelog entry for your changes:

* No changelog required. Merges into `instant-search-master`.
